packages/es-components/src/components/containers/modal/Modal.js

```diff
@@ -105,31 +103,33 @@ function Modal(props) {
     ...other
   } = props;
 
-  const ModalDialog = getModalBySize(size);
   const ariaId = useUniqueId(other.id);
 
+  const shouldCloseOnOverlayClick = backdrop !== 'static' && backdrop !== false;
+
   return (
-    <ModalContext.Provider value={{ onHide, ariaId }}>
-      <BaseModal
-        aria-labelledby={ariaId}
-        backdrop={backdrop}
-        renderBackdrop={backdropProps => <Backdrop {...backdropProps} />}
-        backdropTransition={animation ? BackdropFade : undefined}
-        keyboard={escapeExits}
-        onEnter={onEnter}
-        onExit={onExit}
-        onHide={onHide}
-        show={show}
-        style={modalStyle}
-        transition={animation ? DialogDropIn : undefined}
```

Comment:
We lost part of the modal transition effect. The modal would both drop down/up and fade in/out independently of the backdrop fade. The new dropdown effect is also quite a bit slower.

Reply:
I am pretty sure I fixed this in my latest pushes.

Reply:
Yep, transition looks spot-on now.